.github/workflows/PR_validation.yml

```diff
@@ -230,6 +230,25 @@ jobs:
         if: ${{ always() }}
         uses: ./.github/actions/remove-env
 
+  Orca-Python-Py37-Clean-Env:
```

Comment:
I think later we can name it Orca-Python-Py37
and add Unit test in the name of other jobs